.github/workflows/build.yml

```diff
@@ -166,13 +169,14 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
-          start: yarn emulators:start
+          start: yarn emulators:start; yarn cypress:logger
```

Comment:
I think the separator should be a comma here. I believe the cypress logger might not run at all if a `;` is used.